src/Security/UserProvider.php

```diff
@@ -56,7 +56,7 @@ public function loadUserByUsername($username): UserInterface
                                 $manageGroups[] = $group->getEmail();
                             }
                         }
-                        if (strpos($group->getEmail(), User::SITE_NPH_PREFIX) === 0 || strpos($group->getEmail(), User::READ_ONLY_GROUP) === 0) {
+                        if ((!str_contains($group->getEmail(), User::NPH_ADMIN_GROUP) && strpos($group->getEmail(), User::SITE_NPH_PREFIX) === 0) || strpos($group->getEmail(), User::READ_ONLY_GROUP) === 0) {
```

Comment:
@Ehaic I think it's better to explicitly check if the group is nph site admin. Something like

         $siteId = preg_replace('/@.*$/', '', $group->getEmail());
          if ($siteId !== self::NPH_ADMIN_GROUP) {